src/efs_parser/BaseFile.py

```diff
@@ -220,18 +220,14 @@ def bin_rows(self, data, chr, start, end, columns=None, metadata=None, bins = 40
             if len(temps) > 0:
                 for col in columns:
                     #row[col] = float(np.mean(temps[col].values))
-                    row[col] = np.mean(self.change_to_float(temps[col].values))
+                    #row[col] = np.mean(self.change_to_float(temps[col].values))
+                    values_to_fix = [float(i) for i in temps[col].values]
```

Comment:
this doesn't work ? 

```
temps[col].apply(float)
```

or this ?

```
temps[col].astype(float64)
```

Reply:
These 2 work:
temps[col].apply(float)
temps[col].astype(float)
However, this does work:
temps[col].astype(float64)

Thank you for the hint. I will update the code accordingly

Reply:
ok, Iets use the pandas inbuilt method `astype`

```temps[col].astype(float)```

Reply:
I have committed the change